COPYRIGHT.txt

```diff
@@ -0,0 +1,14 @@
+
+Copyright 2021 The Sigstore Authors.
```

Comment:
2022

Reply:
> The copyright year for the license headers has been set to the year in which the file was first created.

I assumed

Reply:
Ah, I guess this one can be 2022? I'm not sure. I assumed it was just "when was the dir first created"

Reply:
I’d probably go with this year, I don’t know if you can backdate copyright.

Reply:
Perhaps we should just remove the year? https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/